### DISCUSSION TITLE
Show days of previous and next month

### DISCUSSION THREAD

**Discussion by MiDniGG** [easepick/easepick#90] [Ideas]
I absolutely love easepick. It fits in every requirement we had so far!

Only one more thing would be required to achieve perfection.
Please consider to add the days of the previous and next month in light color.

For example:
![image](https://user-images.githubusercontent.com/6615476/177124120-8accf41d-b8e5-4c04-92e6-270e60788d11.png)

**Comment by stell** [comment]
You can do that by adding custom CSS:
```
.calendar>.days-grid>div:nth-child(7n+7),
.calendar>.days-grid>div:nth-child(7n+1) {
    font-weight:700;
    color: blue;
}
```
https://easepick.com/examples/customize.html

**Reply by MiDniGG** [reply]
No, I didn't mean the weekends, I meant the day of the other month, so in the example 26th - 31st May and 1st - 6th July.
At the moment the fields are only `<div class="offset"></div>` so, no data to show/change with plain css.

**Reply by stell** [reply]
I see. Misunderstanding.

**Comment by loratokareva** [comment]
Do you have a solution?
